bibsonomy-webapp-v2/docs/pages/TagCloudPage.md

```
@@ -0,0 +1,165 @@
+```
+page.tagCloud.title
+page.tagCloud.search.placeholder
+page.tagCloud.filters.minFreq
+page.tagCloud.filters.maxCount
+page.tagCloud.filters.sortBy.frequency
+page.tagCloud.filters.sortBy.alphabetical
+page.tagCloud.filters.sortBy.recent
+page.tagCloud.tagCount
+page.tagCloud.popularTags.title
+page.tagCloud.recentTags.title
+page.tagCloud.relatedTags.title
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)